extractor/extractor.py

```diff
@@ -0,0 +1,205 @@
+import re
+from urllib.request import urlopen
+from pprint import pprint
+
+import requests
+import bs4
+from bs4 import BeautifulSoup, Comment
+from PIL import Image
+
+import contents
+
+req_session = ""
+
+
+class Extractor:
+    def __init__(self, URL):
+        self.url = URL
+        self.soup = ''
+        self.contents_list = contents.Contents()
+
+    def parse(self):
+        global req_session
+
+        req_session = requests.Session()
+        file = req_session.get(self.url).text    # To request Html from URL
+
+        # Remove the comment to read local files for testing purpose
+        # file = open('./test_html.html','r').read()
+
+        self.soup = BeautifulSoup(file,'lxml')
+        self.clean_soup()
+
+        self.extract()
+
+    def clean_soup(self):
+        decomposable_tags = self.soup.find_all(['script', 'noscript', 'style', 'header', 'footer', 'meta', 'link'])
+        comments = self.soup.find_all(text=lambda text: isinstance(text, Comment))
+        ads = self.soup.find_all(re.compile('.*'), {'class': re.compile('(^ad-|^ads-)')})
+
+        comments = [comment.extract() for comment in comments]
+        [tag.decompose() for tag in decomposable_tags]
+        [ad.decompose() for ad in ads]
+
+    def extract(self):
```

Comment:
nit: `extract_soup()` or `extract_html()` instead of `extract()`

Reply:
nit: For other 2 methods invokes in here:
`extract_head_element()`
`extract_body_element()`